src/arch/riscv/RiscvTLB.py

```diff
@@ -44,6 +44,7 @@ class RiscvPagetableWalker(ClockedObject):
             "Number of outstanding walks that can be squashed per cycle")
     # Grab the pma_checker from the MMU
     pma_checker = Param.PMAChecker(Parent.any, "PMA Chekcer")
+    pmp = Param.PMP(Parent.any, "PMP")
```

Comment:
I think this makes sense for now, but I'd like to see a change after this one which factors out these permission checks so we don't have do duplicate the code in both the TLB and the PTW.